test/runtime/ct.go

```diff
@@ -133,12 +141,8 @@ var _ = Describe("RuntimeDisabledValidatedConntrackTable", func() {
 		}
 		vm.PolicyDelAll()
 		netcatPort = 11111
-		containers(helpers.Delete)
-		res := vm.SetPolicyEnforcement(helpers.PolicyEnforcementDefault)
-		res.ExpectSuccess("Setting policyEnforcement to default")
 
-		areEndpointsReady := vm.WaitEndpointsReady()
-		Expect(areEndpointsReady).Should(BeTrue(), "Endpoints not ready after timeout")
+		vm.ExecCilium("bpf ct flush global")
```

Comment:
Why do we need to flush conntrack here? Please add a comment indicating why, as this will help those who run these tests understand what is going on.